front/app/component-library/components/CollapsibleContainer/index.tsx

```diff
@@ -102,25 +102,23 @@ const CollapsibleContainer = ({
   };
 
   return (
-    <Box display="flex" flexDirection="column" {...rest}>
-      <Box display="flex" alignItems="center">
-        <TitleButton
-          as="button"
```

Comment:
Didn't understand this as prop, which seems to be a styled components prop. I think we should go for a straight button. The `styled(Box)` in combination with the `as="button"` prop was confusing me a lot.